data_steward/cleaners/clean_cdr_engine.py

```diff
@@ -0,0 +1,70 @@
+import logging
+
+import googleapiclient
+import oauth2client
+from google.appengine.api import app_identity
+
+import bq_utils
+
+LOGGER = logging.getLogger(__name__)
+
+
+def add_console_logging(add_handler):
+    # this config should be done in a separate module, but that can wait
+    # until later.  Useful for debugging.
+    logging.basicConfig(level=logging.DEBUG,
+                        filename='/tmp/cleaner.log',
+                        filemode='a')
+
+    if add_handler:
+        handler = logging.StreamHandler()
+        handler.setLevel(logging.INFO)
+        formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+        handler.setFormatter(formatter)
+        LOGGER.addHandler(handler)
+
+
+def clean_dataset(project=None, dataset=None, rule_query=None):
```

Comment:
To make this work with an iterable list of SQL statements, `rule_query` needs to be an iterable as it used to be.  We want to send a list of work to clean the dataset, not to run a single query.  If we really want to run a single query, we can achieve that by sending a list of one item.  I had suggested deleting lines 73 to the end because it was redundant.  By keeping lines 73 to the end and deleting the original function, we have lost the iterable quality of this function.